assets/xml/objects/object_gi_bomb_2.xml

```diff
@@ -0,0 +1,5 @@
+<Root>
+    <File Name="object_gi_bomb_2" Segment="6">
+        <DList Name="gGIBomb2DL" Offset="0x4B0"/>
```

Comment:
gBombchuGetItemDL

Reply:
if this is chus I recommend changing this yea.
zels name suggestions for the others are also good, but I would shorten GetItem to GI. its a common acronym anyway